apps/atomy-q/API/app/Http/Requests/ManualNormalizationSourceLineRequest.php

```
@@ -24,8 +25,15 @@ public function rules(): array
             'rfq_line_item_id' => ['nullable', 'string'],
-            'note' => ['nullable', 'string', 'max:1000'],
-            'reason' => ['nullable', 'string', 'max:1000'],
+            'note' => ['nullable', 'string', 'max:1000', 'required_if:reason,other'],
+            'reason' => ['required', 'string', Rule::in([
+                'supplier_document_mismatch',
+                'rfq_mapping_incorrect',
+                'quantity_or_uom_correction',
+                'price_correction',
+                'manual_entry_required',
+                'other',
+            ])],
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Validation\Rule' in method 'rules'. (undefined)

(StaticAccess)